ods_ci/tests/Tests/650__distributed_workloads/test-run-kuberay-e2e.robot

```diff
@@ -9,7 +9,7 @@ Resource          ../../../tasks/Resources/RHODS_OLM/install/oc_install.robot
 *** Variables ***
 ${KUBERAY_DIR}            kuberay
 ${KUBERAY_REPO_URL}       %{KUBERAY_REPO_URL=https://github.com/ray-project/kuberay.git}
```

Comment:
Can you rather use https://github.com/red-hat-data-services/kuberay as this is the source code for product?